core/events/tests/test_classic_integration.py

```diff
@@ -30,13 +30,37 @@ def setUp(self):
         """An arXiv user is submitting a new paper."""
```

Comment:
Can you add some language to the docstring that explains what's going on here? This will look pretty strange to someone new to the project, or who hasn't looked at this in a while.